### ISSUE TITLE
Configuring warning/critical thresholds

### ISSUE THREAD

**Issue by nh905** [vivid-planet/munin-php-apc#7]
I was successfully able to setup php_apc_ even though I am running munin 2.0.25.  At the moment I am only capturing usage data.  Is it possible to define warning/critical thresholds for the 'used' and 'fragmented' monitors?  It appears munin is using a new way of passing variables.  I tried the following configuration - the env.url value is being passed but I do not see the other values reflected in the table below the graphs.

Thanks, Norbert

[php_apc_*]
user (username)
env.url http://(site)/apc_info.php?auto
env.used_warning 47185920
env.used_critical 57671680
env.fragmented_warning 5242880
env.fragmented_critical 10485760

**Comment by nsams** [comment]
Sorry I can't help you wirh that, better ask that as a generic muni question on stackoverflow or something...

**Comment by nh905** [comment]
After some experimentation and code reading, I found out that the old way of overriding thresholds via munin.conf still works.  I did not realize that the plugin names need to be fully spelled out.  I had originally tried "php_apc_fragment_percentage.critical 5" which was not successful.  Compounding the problem was that the two 'stock' plugins (cpu and memory) I looked at both set environment variables through the configuration files in /etc/munin/plugin-conf.d rather than using munin.conf overrides.  These plugins are fetching system information during plugin configuration and then using the environment variables to dynamically set thresholds.  If I get some time, I will write up code to set php_apc_ thresholds based on percentages of the total apc cache size.

I also got php_apc_ working with multiple apc caches.  The critical step is to add the cache identifier before the trailing '_'.  A lot of posts about supporting multiple munin plugin instances added the cache identifier after the trailing '_' (such as "/etc/munin/plugins/mysql_m0_bytes") which did not work for php_apc_.
